SU-SWS/jumpstart_ui
# D8CORE-562: Updates to Hero and Card templates

# READY FOR REVIEW

# Summary
- Fixes and updates to templates hero + card
- Dependant on https://github.com/SU-SWS/decanter/pull/513

# Review By (Date)
- October 22nd

# Urgency
- Medium

# Steps to Test

1. Check out this branch
2. Do a full drupal site install of the stanford_profile profile
3. Review home page banner and cards
4. Remove all text from home page banner and review

# Affected Projects or Products
- D8CORE

# Associated Issues and/or People
- https://github.com/SU-SWS/decanter/pull/513

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
